OpenSim/Simulation/Model/Appearance.h

```diff
@@ -33,105 +33,157 @@ namespace OpenSim {
+        "Name of file containing texture. ");
 
     //--------------------------------------------------------------------------
     // CONSTRUCTION
     //--------------------------------------------------------------------------
 public:
-    BaseAppearance() {
-        constructProperties();
+    SurfaceAppearance() {
+        constructProperty_representation(3); // Surface shaded by default
     }
-    virtual ~BaseAppearance() {};
+    virtual ~SurfaceAppearance() {};
 
-private:
-    void constructProperties() {
-        constructProperty_color(SimTK::Vec3(1.0));
-        constructProperty_opacity(1.0);
-        constructProperty_representation(3);
+    bool hasTexture() {
+        return getProperty_texture_file().size() > 0;
     }
-    //==========================================================================
-};  // END of class BaseAppearance
-
+    //=============================================================================
+};  // END of class SurfaceAppearance
 
-class OSIMSIMULATION_API Appearance : public BaseAppearance {
-    OpenSim_DECLARE_CONCRETE_OBJECT(Appearance, BaseAppearance);
+//=============================================================================
+/** CurveAppearance class holds the display attributes  of a piece of Geometry
+displayed in the OpenSim visualizer or GUI.The attributes in this
+class are specific to curves or line drawings so that their thickness, line-style
+etc. can be maintained. Attributes common to all possible geometries that can be 
+rendered (e.g.color, opacity, visibility) are not included
+here but provided by the Appearance class instead.
+*/
+//=============================================================================
+class OSIMSIMULATION_API CurveAppearance : public Object {
+    OpenSim_DECLARE_CONCRETE_OBJECT(CurveAppearance, Object);
 public:
-    //==========================================================================
+    //==============================================================================
     // PROPERTIES
-    //==========================================================================
-    OpenSim_DECLARE_OPTIONAL_PROPERTY(texture_file, std::string,
-        "Name of file containing texture. ");
+    //==============================================================================
+    OpenSim_DECLARE_PROPERTY(thickness, double,
+        "The thickness of lines used to render a curve or a drawing. ");
 
     //--------------------------------------------------------------------------
     // CONSTRUCTION
     //--------------------------------------------------------------------------
 public:
-    Appearance() {
+    CurveAppearance() {
+        constructProperties();
     }
-    virtual ~Appearance() {};
+    virtual ~CurveAppearance() {};
 
-    bool hasTexture() {
-        return getProperty_texture_file().size() > 0;
+private:
+    void constructProperties() {
+        constructProperty_thickness(.05);
```

Comment:
Why is default 0.05? Perhaps scale this property internally so user-facing default is 1.0?